tests/systemtests/protocolpool_test.go

```diff
@@ -0,0 +1,552 @@
+			require.NoError(t, err)
+			return []byte(state)
+		},
+	)
+}
+
+func TestQueryProtocolPool(t *testing.T) {
+	// Scenario:
+	// delegate tokens to validator
+	// check distribution
+
+	sut := systemtests.Sut
+	sut.ResetChain(t)
+
+	// set up gov params so we can pass props quickly
+	modifyGovParams(t)
+
+	cli := systemtests.NewCLIWrapper(t, sut, systemtests.Verbose)
+
+	// add genesis account with some tokens
+	account1Addr := cli.AddKey("account1")
+	sut.ModifyGenesisCLI(t,
+		[]string{"genesis", "add-genesis-account", account1Addr, "1000000000000stake"},
+	)
+
+	sut.StartChain(t)
+
+	// query validator address to delegate tokens
+	rsp := cli.CustomQuery("q", stakingModule, "validators")
+	valAddr := gjson.Get(rsp, "validators.#.operator_address").Array()[0].String()
+
+	// get validator address
+	valSigner := gjson.Get(cli.Keys("keys", "list"), "0.address").String()
+	require.NotEmpty(t, valSigner)
+
+	// stake tokens
+	rsp = cli.Run("tx", stakingModule, "delegate", valAddr, "10000000000stake", "--from="+account1Addr, "--fees=1stake")
+	systemtests.RequireTxSuccess(t, rsp)
+
+	beforeBalance := cli.QueryBalance(account1Addr, stakingToken)
+	assert.Equal(t, int64(989999999999), beforeBalance)
+
+	rsp = cli.CustomQuery("q", stakingModule, "delegation", account1Addr, valAddr)
+	assert.Equal(t, "10000000000", gjson.Get(rsp, "delegation_response.balance.amount").String(), rsp)
+	assert.Equal(t, stakingToken, gjson.Get(rsp, "delegation_response.balance.denom").String(), rsp)
+
+	t.Run("check x/distribution query does not work when using x/protocolpool", func(t *testing.T) {
+		failingCli := cli.WithRunErrorMatcher(func(t assert.TestingT, err error, msgAndArgs ...interface{}) (ok bool) {
+			assert.Error(t, err)
+			return false
+		})
+		// query the community pool - should fail for x/distribution
+		rsp = failingCli.CustomQuery("q", distributionModule, "community-pool")
+	})
+
+	t.Run("check x/protocolpool community pool query", func(t *testing.T) {
+		// query will work for x/protocolpool
+		rsp = cli.CustomQuery("q", protocolPoolModule, "community-pool")
+		poolAmount := gjson.Get(rsp, "pool.0.amount").Int()
+		assert.True(t, poolAmount > 0, rsp)
+		assert.Equal(t, stakingToken, gjson.Get(rsp, "pool.0.denom").String(), rsp)
+
+		t.Log("block height", sut.CurrentHeight(), "\n")
+		sut.AwaitNBlocks(t, 1)
+		t.Log("block height", sut.CurrentHeight(), "\n")
+
+		rsp = cli.CustomQuery("q", protocolPoolModule, "community-pool")
+		newPoolAmount := gjson.Get(rsp, "pool.0.amount").Int()
+		assert.Equal(t, stakingToken, gjson.Get(rsp, "pool.0.denom").String(), rsp)
+		// check that staking is continually rewarded
+		assert.True(t, newPoolAmount > poolAmount, rsp)
+	})
+
+	rsp = cli.CustomQuery("q", protocolPoolModule, "community-pool")
+	poolAmount := gjson.Get(rsp, "pool.0.amount").Int()
+	assert.True(t, poolAmount > 0, rsp)
+	assert.Equal(t, stakingToken, gjson.Get(rsp, "pool.0.denom").String(), rsp)
```

Comment:
seems like some duplicate behavior? is there a reason to do this again outside the subtest?